src/sanitation.js

```diff
@@ -247,18 +280,22 @@ export function isValidAttr(
   }
 
   // Remove blacklisted attributes from specific tags e.g. input[formaction].
-  const attrBlacklist = isAmp4Email(doc)
-    ? EMAIL_BLACKLISTED_TAG_SPECIFIC_ATTRS[tagName]
-    : BLACKLISTED_TAG_SPECIFIC_ATTRS[tagName];
+  const attrBlacklist = Object.assign(
```

Comment:
can we create a private blacklistedTags_ call which the blacklistedTags will call. Use private call here and below

e.g.

const blacklistedTags_ = getBlacklistedTags_.bind(null, isAmp4Email(doc));

export function blacklistedTags(doc) {
  return  blacklistedTags_(BLACKLISTED_TAGS, EMAIL_BLACKLISTED_TAGS);
}

function getBlacklistedTags_(isAmp4Email, blacklistTarget, blacklistSource) {
  return Object.assign(map(), blacklistTarget, isAmp4Email ? blacklistSource {});
}

const attrBlacklist = blacklistTags_(BLACKLISTED_TAG_SPECIFIC_ATTRS, 
    EMAIL_BLACKLISTED_TAG_SPECIFIC_ATTRS);

const attrValueBlacklist = blacklistTags_(
    BLACKLISTED_TAG_SPECIFIC_ATTR_VALUES,
    EMAIL_BLACKLISTED_TAG_SPECIFIC_ATTR_VALUES)[tagName];

Reply:
Is the intent to have code reuse for the `Object.assign` call? It seems harder to read IMO.

Reply:
Yes, I don't feel strongly about it so that's fine. I'll look over the rest of the changes before end of the day. Thanks